api/src/middlewares/services/UserService/personalisation/personalisationBuilder.userPostalConfirmation.ts

```diff
@@ -19,6 +19,10 @@ export function buildUserPostalConfirmationPersonalisation(answers: AnswersHashM
   const post = answers["post"] as string;
   const previousMarriage = answers.maritalStatus && answers.maritalStatus !== "Never married";
 
+  // Italy is the only country that requires the partner's proof of end of marriage doc
+  const partnerHasPreviousMarriage = answers.partnerMaritalStatus && answers.partnerMaritalStatus !== "Never married";
```

Comment:
do we need to make a notify change during release then? (probably easier to just copy the template and change the env var when releasing?)

Reply:
Luckily since I accidentally updated the template yesterday and then reverted it's been added as a previous version. I've created a reminder to revert back to that version when release day comes.